dev/io.openliberty.security.jakartasec.3.0.internal/src/io/openliberty/security/jakartasec/identitystore/OpenIdContextImpl.java

```diff
@@ -233,7 +233,7 @@ public <T> Optional<T> getStoredValue(HttpServletRequest request, HttpServletRes
             value = (T) storage.get(storageName);
         }
 
-        return Optional.of(value);
+        return value != null ? Optional.of(value) : Optional.empty();
```

Comment:
shorthand for what you are doing:
```suggestion
        return Optional.ofNullable(value);
```

Reply:
thanks